django/contrib/gis/db/backends/postgis/base.py

```diff
@@ -24,3 +24,6 @@ def prepare_database(self):
         # Check that postgis extension is installed.
         with self.cursor() as cursor:
             cursor.execute("CREATE EXTENSION IF NOT EXISTS postgis")
+        if self.ops.postgis_version_tuple()[1:] >= (3, 0, 0):
+            with self.cursor() as cursor:
+                cursor.execute('CREATE EXTENSION IF NOT EXISTS postgis_raster')
```

Comment:
Might make sense to make it optional. Should we add setting for this?

Reply:
Definitely worth making it optional for those who don't (want to) use any of the raster bits.

I don't know whether we should put it behind an option or get people to just use `CreateExtension` instead. Could then be a bit of documentation to explain what pieces need the extension and how to enable it, and a warning in the release notes that users upgrading to PostGIS 3.0 will need to enable the extension...